### PR TITLE
DOCSP-41971: Bulk write

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-php-library/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-41971
Staging - https://preview-mongodbnorareidy.gatsbyjs.io/php-library/DOCSP-41971-bulk/write/bulk-write/

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [ ] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/write/bulk-write.txt [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+       [ 'insertOne'  => [ $document ] ],
+       [ 'replaceOne' => [ $filter, $replacement, $options ] ],
+       [ 'updateMany' => [ $filter, $update, $options ] ],
+       [ 'updateOne'  => [ $filter, $update, $options ] ],
+   ]
+
+.. tip::
+
+    For more information about delete, insert, replace, and update
+    operations, see the :ref:`Write operation guides <php-write>`.
```
Comment (norareidy) [inline source/write/bulk-write.txt:71]: Note to reviewer: this landing page will eventually include usage examples for each write operation

File: source/write/bulk-write.txt [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+Additional Information
+----------------------
+
+To learn how to perform individual write operations, see the following guides:
+
+- :ref:`php-write-insert`
+- :ref:`php-write-update`
+- :ref:`php-write-delete`
+- :ref:`php-write-replace`
```
Comment (norareidy) [inline source/write/bulk-write.txt:220]: Note to reviewer: link will be broken until #125 is merged

File: source/write/bulk-write.txt [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+
+   * - Option
+     - Description
+
+   * - ``bypassDocumentValidation``
+     - | Specifies whether the operation bypasses document validation. This lets you 
+         modify documents that don't meet the schema validation requirements, if any 
+         exist. For more information about schema validation, see :manual:`Schema
+         Validation </core/schema-validation/#schema-validation>` in the MongoDB
+         Server manual.
```
Comment (jordan-smith721) [inline source/write/bulk-write.txt:119]:
```suggestion
         Validation </core/schema-validation/#schema-validation>` in the {+mdb-server+} manual.
```

File: source/write/bulk-write.txt [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+         Values must be constant or closed expressions that don't reference document
+         fields. For more information, see the :manual:`let statement
+         </reference/command/update/#std-label-update-let-syntax>` in the
+         {+mdb-server+} manual.
+
+   * - ``ordered``
+     - | If set to ``true``: when a single write fails, the operation stops without
+         performing the remaining writes and throws an exception.
+       | If set to ``false``: when a single write fails, the operation continues to
+         attempt the remaining write operations, if any, and throws an exception.
```
Comment (jordan-smith721) [inline source/write/bulk-write.txt:143]: S: I think it throws the exception after attempting the other operations?

```suggestion
         attempt the remaining write operations, if any, then throws an exception.
```

**Review by jordan-smith721 — APPROVED** [review]
A few suggestions but otherwise LGTM!

**Review by GromNaN — APPROVED** [review]

**Pull request merged**